rln/src/circuit.rs

```diff
@@ -22,23 +21,24 @@ cfg_if! {
         use once_cell::sync::OnceCell;
         use std::sync::Mutex;
         use wasmer::{Module, Store};
+        use include_dir::{include_dir, Dir};
     }
 }
 
 const ZKEY_FILENAME: &str = "rln_final.zkey";
-const VK_FILENAME: &str = "verifying_key.json";
+const VK_FILENAME: &str = "verification_key.json";
 const WASM_FILENAME: &str = "rln.wasm";
 
 // These parameters are used for tests
 // Note that the circuit and keys in TEST_RESOURCES_FOLDER are compiled for Merkle trees of height 15, 19 and 20
 // Changing these parameters to other values than these defaults will cause zkSNARK proof verification to fail
 pub const TEST_PARAMETERS_INDEX: usize = 2;
 pub const TEST_TREE_HEIGHT: usize = [15, 19, 20][TEST_PARAMETERS_INDEX];
-pub const TEST_RESOURCES_FOLDER: &str = [
-    "./resources/tree_height_15/",
-    "./resources/tree_height_19/",
-    "./resources/tree_height_20/",
-][TEST_PARAMETERS_INDEX];
+pub const TEST_RESOURCES_FOLDER: &str =
+    ["tree_height_15/", "tree_height_19/", "tree_height_20/"][TEST_PARAMETERS_INDEX];
```

Comment:
These slashes should be deleted

Reply:
Addressed in [cace359](https://github.com/vacp2p/zerokit/pull/118/commits/cace35911dbc9642367b13695c968ba575249c29)